File: Desktop/PIDEV_JavaFX/src/web_java/sujets_forum.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Web_java;
import java.util.Date;

/**
 *
 * @author DELL
 */
 public class sujets_forum {
    private long id_sujet_forum;
    private Date date_creations;
    private String sujet;
    private String theme;
    private String descriptif_sujet;

    public sujets_forum(long id_sujet_forum) {
        this.id_sujet_forum = id_sujet_forum;
    }

    public sujets_forum() {
    }
    

    public sujets_forum(long id_sujet_forum, Date date_creations) {
        this.id_sujet_forum = id_sujet_forum;
        this.date_creations = date_creations;
    }
    
    
    

    public sujets_forum(long id_sujet_forum, Date date_creations, String sujet, String theme, String descriptif_sujet) {
        this.id_sujet_forum = id_sujet_forum;
        this.date_creations = date_creations;
        this.sujet = sujet;
        this.theme = theme;
        this.descriptif_sujet = descriptif_sujet;
    }

    public long getId_sujet_forum() {
        return id_sujet_forum;
    }

    public void setId_sujet_forum(long id_sujet_forum) {
        this.id_sujet_forum = id_sujet_forum;
    }

    public Date getDate_creations() {
        return date_creations;
    }

    public void setDate_creations(Date date_creations) {
        this.date_creations = date_creations;
    }

    public String getSujet() {
        return sujet;
    }

    public void setSujet(String sujet) {
        this.sujet = sujet;
    }

    public String getTheme() {
        return theme;
    }

    public void setTheme(String theme) {
        this.theme = theme;
    }

    public String getDescriptif_sujet() {
        return descriptif_sujet;
    }

    public void setDescriptif_sujet(String descriptif_sujet) {
        this.descriptif_sujet = descriptif_sujet;
    }
    
    

    

    

    
    
    
}

File: mysymfony2/IRON_FIT_Front-PHP-Symfony-main/src/Entity/SujetsForum.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\SujetForumRepository;

/**
 * @ORM\Entity
 * @ORM\Table(name="sujets_forum")
 */
#[ORM\Entity]
#[ORM\Table(name: "sujets_forum")]
class SujetsForum
{
    #[ORM\Id]
    #[ORM\GeneratedValue(strategy: "AUTO")]
    #[ORM\Column(type: "bigint")]
    private $idSujetForum;

    #[ORM\Column(type: "date")]
    private $dateCreations;

    #[ORM\Column(type: "string", length: 255)]
    private $sujet;

    #[ORM\Column(type: "string", length: 255)]
    private $theme;

    #[ORM\Column(type: "text", nullable: true)]
    private $descriptifSujet;

    // Getters and Setters

    public function getIdSujetForum(): ?string
    {
        return $this->idSujetForum;
    }

    public function setIdSujetForum(?string $idSujetForum): self
    {
        $this->idSujetForum = $idSujetForum;
        return $this;
    }

    public function getDateCreations(): ?\DateTimeInterface
    {
        return $this->dateCreations;
    }

    public function setDateCreations(\DateTimeInterface $dateCreations): self
    {
        $this->dateCreations = $dateCreations;
        return $this;
    }

    public function getSujet(): ?string
    {
        return $this->sujet;
    }

    public function setSujet(string $sujet): self
    {
        $this->sujet = $sujet;
        return $this;
    }

    public function getTheme(): ?string
    {
        return $this->theme;
    }

    public function setTheme(string $theme): self
    {
        $this->theme = $theme;
        return $this;
    }

    public function getDescriptifSujet(): ?string
    {
        return $this->descriptifSujet;
    }

    public function setDescriptifSujet(?string $descriptifSujet): self
    {
        $this->descriptifSujet = $descriptifSujet;
        return $this;
    }
}
